apps/coa/translations/src/en/pages.json

```diff
@@ -14,7 +14,10 @@
     "intro": "For security, you must provide us with information from an identity document that UK Visas and Immigration has on {{#values.isApplicant}}your{{/values.isApplicant}}{{^values.isApplicant}}{{values.nameWithPossession}}{{/values.isApplicant}} record."
   },
   "upload-identity": {
-    "header": "Upload proof of identity"
+    "header": "Upload proof of identity",
+    "upload-identity-intro": "For security, you must prove {{#isApplicant}}your{{/isApplicant}}{{^isApplicant}}{{values.applicant-full-name}}’s{{/isApplicant}} identity.",
```

Comment:
Replace the condition check with below as the current check always execute the false condition.

`{{#values.isApplicant}}your{{/values.isApplicant}}{{^values.isApplicant}}{{values.nameWithPossession}}{{/values.isApplicant}} `